Sources/OutlineView/OutlineView.swift

```diff
@@ -48,34 +165,79 @@ where Data.Element: Identifiable {
     ///
     /// - Parameters:
     ///   - data: A collection of tree-structured, identified data.
+    ///   - selection: A binding to a selected value.
     ///   - children: A key path to a property whose non-`nil` value gives the
     ///     children of `data`. A non-`nil` but empty value denotes an element
     ///     capable of having children that's currently childless, such as an
     ///     empty directory in a file system. On the other hand, if the property
-    ///     at the key path is `nil`, then the outline group treats `data` as a
+    ///     at the key path is `nil`, then the outline view treats `data` as a
     ///     leaf in the tree, like a regular file in a file system.
-    ///   - selection: A binding to a selected value.
     ///   - content: A closure that produces an `NSView` based on an
     ///     element in `data`. An `NSTableCellView` subclass is preferred.
     ///     The `NSView` should return the correct `fittingSize`
     ///     as it is used to determine the height of the cell.
-    public init(
+    init(
         _ data: Data,
+        selection: Binding<Data.Element?>,
         children: KeyPath<Data.Element, Data?>,
```

Comment:
Sorry, that I wasn't explicit about this. While I like the symmetry b/w the two variants of the inits, I think in this particular case keeping the existing order of parameters for the KeyPath variant should take priority.

Even though this will be a major version bump, keeping the parameter order lets the users update with fewer (or no) code changes.

Can you please move the children parameter above the selection?

Reply:
Oh yeah. That makes sense. Fixed here: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/2a4af9d9a0c5ef022b58946da454419c9b3f61d3

Reply:
I would expect the other two initializers for macOS 11 and macOS 11 + No Drop to also be changed back to the same order. This ensures that if you have existing code, applying a drop receiver will not cause a build failure because of the wrong order of the parameters.

The beauty of having closures with the changes that you have added is that even if they use the `children` parameter in the new position, they can still supply a key path! :)

Please update the parameter order in the doc comments as well.

Reply:
Doh! That's what I get for rushing. Updated a few of them, but not all. Just pushed one last (hopefully) commit with the doc comments and init parameters reordered.